File: ClinicDesktop/untitled/src/com/example/clinicsystem/code/models/classes/M_PrescriptionInstruction.java
package com.example.clinicsystem.code.models.classes;

public class M_PrescriptionInstruction {
    private int drugID, quantity, duration, dose;
    private String durationType, instructionDetails;

    public M_PrescriptionInstruction(int drugID, int duration, int dose, String durationType, String instructionDetails, int quantity) {
        this.drugID = drugID;
        this.duration = duration;
        this.dose = dose;
        this.durationType = durationType;
        this.instructionDetails = instructionDetails;
        this.quantity = quantity;
    }

    public M_PrescriptionInstruction() {
    }

    public int getDrugID() {
        return drugID;
    }

    public void setDrugID(int drugID) {
        this.drugID = drugID;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public int getDose() {
        return dose;
    }

    public void setDose(int dose) {
        this.dose = dose;
    }

    public String getDurationType() {
        return durationType;
    }

    public void setDurationType(String durationType) {
        this.durationType = durationType;
    }

    public String getInstructionDetails() {
        return instructionDetails;
    }

    public void setInstructionDetails(String instructionDetails) {
        this.instructionDetails = instructionDetails;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: clinic/M_PrescriptionInstruction.php
<?php
class M_PrescriptionInstruction {
    private $drugID, $quantity, $duration, $dose;
    private $durationType, $instructionDetails;

    public function __construct($drugID,$duration,$dose,$durationType, $instructionDetails, $quantity) {
        $this->drugID =             $drugID;
        $this->duration =           $duration;
        $this->dose = 	  	        $dose;
        $this->durationType =       $durationType;
        $this->instructionDetails = $instructionDetails;
        $this->quantity = 			$quantity;
    }

    public function getDrugID() {
        return $this->drugID;
    }

    public function setDrugID($drugID) {
        $this->drugID = $drugID;
    }

    public function getDuration() {
        return $this->duration;
    }

    public function setDuration($duration) {
        $this->duration = $duration;
    }

    public function getDose() {
        return $this->dose;
    }

    public function setDose($dose) {
        $this->dose = $dose;
    }

    public function getDurationType() {
        return $this->durationType;
    }

    public function setDurationType($durationType) {
        $this->durationType = $durationType;
    }

    public function getInstructionDetails() {
        return $this->instructionDetails;
    }

    public function setInstructionDetails($instructionDetails) {
        $this->instructionDetails = $instructionDetails;
    }

    public function getQuantity() {
        return $this->quantity;
    }

    public function setQuantity($quantity) {
        $this->quantity = $quantity;
    }
?>